manifests/client/configure.pp

```diff
@@ -35,8 +34,6 @@
     $lbclient_cert_file_path = undef,
     $lbclient_cert_password = undef,
     $lbclient_cert_alias = undef,
-    $before = [],
```

Comment:
That line causes the syntax check to fail. Apparently $before in that context is implicit and so declaring it like this is unnecessary.